bigquery/main.go

```diff
@@ -26,10 +26,15 @@ import (
 	"cloud.google.com/go/civil"
 	"github.com/GoogleCloudPlatform/cloud-build-notifiers/lib/notifiers"
 	log "github.com/golang/glog"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/v1/google"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
 	cbpb "google.golang.org/genproto/googleapis/devtools/cloudbuild/v1"
 )
 
 var tableResource = regexp.MustCompile(".*/.*/.*/(.*)/.*/(.*)")
+var terminalStatusCodes = []string{"SUCCESS", "FAILURE"}
```

Comment:
We probably want to list all of them (e.g. TIMEOUT, CANCELLED). It's fine to leave this as a TODO for now.